exampleCourse/courseInstances/Sp15/assessments/homework1/infoAssessment.json

```diff
@@ -22,7 +22,8 @@
                 {"id": "partialCredit3",            "points": 3, "maxPoints": 11, "triesPerVariant": 5},
                 {"id": "partialCredit4_v2",         "points": 4, "maxPoints": 13},
                 {"id": "partialCredit5_v2_partial", "points": 5, "maxPoints": 12},
-                {"id": "partialCredit6_no_partial", "points": 3, "maxPoints": 8}
+                {"id": "partialCredit6_no_partial", "points": 3, "maxPoints": 8},
+                {"id": "generatedMultipleChoiceR",  "points": 3, "maxPoints": 11, "triesPerVariant": 5}
```

Comment:
This is causing the test suite to fail, because it uses `homework1` for testing and assumes a particular structure for it (number of total points, etc). The easiest way to fix this is to delete `generatedMultipleChoiceR` from `homework1`. You can add `generatedMultipleChoiceR` to `homework2` if you want it in an assessment (not required, but fine either way).

Can you also add `generatedMultipleChoiceR` to the `qids` array in `tests/testQuestions.js` so that it gets auto tested?

Reply:
Ah! Thanks for pointing this out. I've rectified it in: https://github.com/PrairieLearn/PrairieLearn/pull/1023/commits/f605ee81c733a462716850a620efe0ace0cd8326